libbzip2-rs-sys/src/bzlib.rs

```diff
@@ -1227,10 +1233,12 @@ unsafe fn un_rle_obuf_to_output_fast(strm: &mut bz_stream, s: &mut DState) -> bo
                     if c_state_out_len == 1 {
                         break;
                     }
-                    *(cs_next_out as *mut u8) = c_state_out_ch;
+                    unsafe {
+                        *(cs_next_out as *mut u8) = c_state_out_ch;
+                    }
```

Comment:
if you move the  `;` to after the `}` of the unsafe block, this formats on one line